app/config/default.json

```diff
@@ -1,4 +1,5 @@
 {
+    "api": "https://www.targetvalidation.org/api",

```

Comment:
i believe the webapp queries for version (eg `latest`) after this URL. With this PR, we could start leaving the version logic in the api (eg. `mycrazyversion.api.opentargets.io` or `master.api.opentargets.io`) and just point to the right version for each branch using `APIHOST`